tests/bluetooth/audio/ascs/src/test_common.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/fff.h>
+#include <zephyr/types.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/sys/util_macro.h>
+
+#include "bap_unicast_server.h"
+#include "bap_stream.h"
+#include "gatt_expects.h"
+
+#include "test_common.h"
+
+static uint8_t attr_found(const struct bt_gatt_attr *attr, uint16_t handle, void *user_data)
+{
+	const struct bt_gatt_attr **result = user_data;
+
+	*result = attr;
+
+	return BT_GATT_ITER_STOP;
+}
+
+const struct bt_gatt_attr *test_ase_control_point_get(void)
+{
+	static const struct bt_gatt_attr *attr;
+
+	if (attr == NULL) {
+		bt_gatt_foreach_attr_type(0x0001, 0xFFFF, BT_UUID_ASCS_ASE_CP, NULL, 1,
+					  attr_found, &attr);
```

Comment:
Since we are including `gatt.h` which includes `att.h` we could use `BT_ATT_FIRST_ATTRIBUTE_HANDLE` and `BT_ATT_LAST_ATTRIBUTE_HANDLE` here